fdbclient/include/fdbclient/ServerKnobs.h

```diff
@@ -591,6 +591,8 @@ class ServerKnobs : public KnobsImpl<ServerKnobs> {
 	// Use global tag throttling strategy. i.e. throttle based on the cluster-wide
 	// throughput for tags and their associated quotas.
 	bool GLOBAL_TAG_THROTTLING;
+	// Enforce tag throttling on proxies rather than on clients
+	bool ENFORCE_TAG_THROTTLING_ON_PROXIES;
```

Comment:
Want's the motivation to throttle on proxies than on clients? Are you plan to also include this change in this status report PR?